### PR TITLE
Microsoft Security Advisory 3137909

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Updates recommended as specified here: https://technet.microsoft.com/library/security/3137909

@HenrikFrystykNielsen

### CODE REVIEW

**Comment by HenrikFrystykNielsen** [comment]
@JunTaoLuo, the PR has been committed, see 53d6efb7826ae416d694c3c966c4a6ced69cb451. 

Would you mind deleting the server johluo/security-update branch? 

Thanks! Henrik

File: samples/CustomSender/Views/Manage/Index.cshtml [hunk 1]
```diff
@@ -39,18 +39,17 @@
...
-                    @Html.ActionLink("Change", "AddPhoneNumber")
-                    @: &nbsp;|&nbsp;
-                    @Html.ActionLink("Remove", "RemovePhoneNumber")
-                }
-                else
-                {
-                    @Html.ActionLink("Add", "AddPhoneNumber")
-                }
-                ]
+                 @(If(Model.PhoneNumber, "None"))
```
Comment (Eilon) [inline samples/CustomSender/Views/Manage/Index.cshtml]: Ummm why did this C# code change into VB??

Comment (Eilon) [inline samples/CustomSender/Views/Manage/Index.cshtml]: @JunTaoLuo @HenrikFrystykNielsen  ping to make sure you see this...

**Comment by Eilon** [comment]
Ok apparently this is an error on https://technet.microsoft.com/library/security/3137909, which @blowdart is going to get fixed.

**Comment by JunTaoLuo** [comment]
Oops, I remember copying that code without looking in at least another repo, going to hunt it down.

**Pull request closed without merging**